pkgs/top-level/all-packages.nix

```
@@ -19291,7 +19291,13 @@ with pkgs;
 
   atkmm_2_36 = callPackage ../development/libraries/atkmm/2.36.nix { };
 
-  pixman = callPackage ../development/libraries/pixman { };
+  pixman = callPackage ../development/libraries/pixman (
+    # https://gitlab.freedesktop.org/pixman/pixman/-/issues/46
+    lib.optionalAttrs stdenv.hostPlatform.isAarch64 {
+      # Just gccStdenv won't work, as on aarch64-darwin it still uses clang's assembler.
+      stdenv = stdenv.override { cc = buildPackages.gcc; };
```

This should take care of the requisites problem:

```suggestion
      stdenv = overrideCC stdenv buildPackages.gcc;
```

---

Hmm right, thanks.  Sounds better than the more complex variant I wrote.